src/main/kotlin/io/spine/publishing/gradle/GradleProject.kt

```diff
@@ -36,7 +36,7 @@ data class GradleProject(private val rootDir: Path) {
     }
 
     /**
-     * Runs the `build` task on this project.
+     * Runs the `build -x checkVersionIncrement` task on this project.
```

Comment:
Do we always run this? I thought it is only needed for the libs which versions aren't bumped.